app/src/main/res/layout/popup_window_browser_menu.xml

```diff
@@ -36,7 +36,9 @@
             android:layout_weight="1"
             android:contentDescription="@string/back"
             android:onClick="onGoBackClicked"
-            android:src="@drawable/ic_arrow_back_24px" />
+            android:src="@drawable/ic_arrow_back_24px"
+            android:tint="@color/popup_menu_buttons_tint"
```

Comment:
The tint was defined in the shared `BrowserIconMenuItem` style and point to an existing selectable drawable. Let's remove these duplicates and update `browser_menu_icon`. I think you may be able to delete the paleOrange color from colors now too.

Reply:
ah thanks! that explains why I couldn't see it.